src/fairseq2/checkpoint/_manager.py

```diff
@@ -980,6 +980,8 @@ def get_step_numbers(self, *, exclude_model_only: bool = False) -> list[int]:
                     # model.
                     if self._file_system.exists(trainer_dir):
                         step_numbers.append(step_nr)
+                else:
+                    step_numbers.append(step_nr)
```

Comment:
? is it needed ? was it a bug ?

Reply:
Added in the fs2:v0.5, can be omitted from up-level.
 https://github.com/facebookresearch/fairseq2/blob/v0.5.0a1/src/fairseq2/checkpoint/manager.py#L769